website/docs/releases/2024/v2024.8.md

```diff
@@ -70,6 +70,14 @@ To try out the release candidate, replace your Docker image tag with the latest
 
     It is now possible to configure a SAML Source to decrypt and validate encrypted assertions. This can be configured by certaing a [Certificate-keypair](../../core/certificates.md) and selecting it in the SAML Source.
 
+-   **Removal of LDAP Provider search group**
+
+    The LDAP provider now uses RBAC to assign the permission to search the full directory instead of requiring a dedicated group to be created. As part of the upgrade, existing search groups' users are migrated to grant the required permission to continue searching the full directory.
```

Comment:
```suggestion
    The LDAP provider now uses RBAC to assign the permission to search the full directory instead of requiring a dedicated group to be created. As part of the upgrade, existing search groups' users are migrated to grant the required permission to search the full directory.
```